libs/design/src/molecules/list/public_api.ts

```diff
@@ -1,4 +1,4 @@
 export { DaffListModule } from './list.module';
-export * from './list/list.component';
-export * from './list-subheader/list-subheader.directive';
-export * from './list-item/list-item.component';
+export { DaffListComponent } from './list/list.component';
```

Comment:
These fundamentally change the API, which I believe is outside the scope of this PR.

Reply:
Do you want me to split this in a separate PR then? I recall you saying that we should be exporting them this way, but I'm not sure about the reasoning.

Reply:
Yes. I would just not change this file for the time being until we decide on what the actual public api should be.

Reply:
reverted